opendevin/config.py

```diff
@@ -40,47 +41,19 @@
         config[k] = tomlConfig[k]
 
 
-def _get(key: str, default):
-    value = config.get(key, default)
-    if not value:
-        value = os.environ.get(key, default)
-    return value
-
-
-def get_or_error(key: str):
+def get(key: str, required: bool = False):
     """
-    Get a key from the config, or raise an error if it doesn't exist.
+    Get a key from the environment variables or config.toml or default configs.
     """
-    value = get_or_none(key)
-    if not value:
+    value = config.get(key)
+    if not value and required:
```

Comment:
Looks like we lost `environ.get` here

```suggestion
    value = os.environ.get(key)
    if not value:
      value = config.get(key)
    if not value and required:
```

Reply:
No

Reply:
s. we don't need lines 37 to 41 too

Reply:
Just saw this--you're totally right! Maybe we can refactor in another PR